foxy/distribution.yaml

```diff
@@ -3083,6 +3083,24 @@ repositories:
       url: https://github.com/ros-controls/realtime_tools.git
       version: foxy-devel
     status: maintained
+  repository_name:
+    doc:
+      type: git
+      url: https://github.com/OUXT-Polaris/lua_vendor.git
+      version: 0.0.1
+    release:
+      packages:
+      - lua_vendor
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/OUXT-Polaris/lua_vendor-release.git
+      version: 0.0.1-1
+    source:
+      test_pull_requests: true
```

Comment:
@hakuturu583 just in case, have you [configured your repository](http://wiki.ros.org/buildfarm/Pull%20request%20testing) for this to work?